apps/ws/src/SocketManager.ts

```diff
@@ -37,7 +37,7 @@ export class SocketManager {
       ...(this.interestedSockets.get(roomId) || []),
       user,
     ]);
-    this.userRoomMappping.set(user.id, roomId);
+    this.userRoomMappping.set(user.userId, roomId);
```

Comment:
Also, a new id was being created which wasn't able to uniquely identify the user.

Reply:
Ah this might be it I guess